app/scripts/components/common/mapbox/aoi/mb-aoi-draw.js

```diff
@@ -2,9 +2,7 @@ import { useEffect, useRef } from 'react';
 import MapboxDraw from '@mapbox/mapbox-gl-draw';
 import { css } from 'styled-components';
 
-import * as RestrictedRectangleMode from './restricted-rect-mode';
```

Comment:
This file doesn't seem to be used any more. Do we want to delete this file?

Reply:
Sorry, forgot it there